redisai/commands_test.go

```diff
@@ -467,6 +467,50 @@ func TestCommand_ModelSet(t *testing.T) {
 	}
 }
 
+func TestCommand_ModelStore(t *testing.T) {
+
+	keyModelStore1 := "test:ModelStore:1"
+	keyModelStore1Pipelined := "test:ModelStore:2:Pipelined"
+	keyModelStoreUnexistant := "test:ModelStore:3:Unexistant"
+	dataUnexistant := []byte{}
+	data, err := ioutil.ReadFile("./../tests/test_data/creditcardfraud.pb")
+	if err != nil {
+		t.Errorf("Error preparing for ModelStore(), while reading file. error = %v", err)
+		return
+	}
+
+	type args struct {
+		name            string
+		backend         string
+		device          string
+		tag             string
+		batchsize       int64
+		minbatchsize    int64
+		minbatchtimeout int64
+		data            []byte
+		inputs          []string
+		outputs         []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{keyModelStore1, args{keyModelStore1, BackendTF, DeviceCPU, "", 0, 0, 0, data, []string{"transaction", "reference"}, []string{"output"}}, false},
```

Comment:
Please add a short documentation for every test case

Reply:
It's hard to know here :sweat_smile:
I copied this from the `AI.MODELSET` test